apache/superset
# chore: type FilterableTable

### SUMMARY
As titled. Related to #10004 

### TEST PLAN
- CI
- run a query in sql lab and see the results
- run a query with a very wide table and see the column widths set properly and scrolling working
- run a query with json object results and see clicking on the results to expand the blob working

### ADDITIONAL INFORMATION



to: @ktmud @nytai @graceguo-supercat
